pyopenms_viz/_core.py

```
@@ -14,6 +14,7 @@
 from pandas.util._decorators import Appender
 import re
 
+
 from numpy import ceil, log1p, log2, nan, mean, repeat, concatenate
```

Ruff (0.8.2)

`numpy.log2` imported but unused

Remove unused import

(F401)